### ISSUE TITLE
Subpoenas and the right to not appear in court

### ISSUE THREAD

**Issue by yopdog** [liberland/constitution#343]
I think subpoenas should be outlawed. I see no justification for forcing an innocent person who is not even suspected of a crime to testify or hand over his property as evidence. I know that the goal of subpoenas is to ensure justice in the court, but the right of the defendant (or prosecutor) to gather evidence in their favour should end where the rights of others begin. A person should not be considered the property of the court simply for being a witness to a crime, the same goes for his property.

The second change I would make would be to ensure that defendants have the right to not appear in court if that's what they want. Being present in court should not only be a right but also a choice. I therefore suggest we make the change to "one's right to **choose** to be present in Court".  

Thirdly, I would like to change this sentence: 
>"to not be compelled to be a witness against oneself, or Persons whose penalization one would rightfully feel as the penalization of oneself"

to:

>"to not be compelled to be a witness against oneself, or anybody else"   

   
So these would be the changes to provision §II.18
       
>§II.18. No law shall deprive a defendant under prosecution of one's right to **choose** to be present in Court; to a speedy, public and impartial trial by the Jury; to be informed of criminal charges and of one's rights; ~~to compel witnesses to appear in Court for confrontation;~~ to be presumed innocent until proven guilty beyond reasonable doubt; to not be compelled to be a witness against oneself, **or anybody else**; and to all other Rights guaranteed by the Constitution and all laws passed in accordance thereof; should any Person be prosecuted in breach of due process of law as enumerated in this provision, the trial Judges shall take that under advisement when considering dismissing the charges or overturning the verdict.

To provision §I.10. I would like to add the following line: "**Warrants shall be issued solely against persons who can reasonably be suspected to have been involved in an offence**". This will ensure that warrants can not be used to collect evidence from non-involved parties, such as witnesses, as would otherwise be allowed without this change.        

>§I.10. All Warrants in the Free Republic of Liberland, other than the Supreme Warrant, shall be issued by a Judge of the Criminal Court upon request by an Agent of the Law Enforcement and shall be supported by evidence of probable cause; such Warrants shall be as detailed as possible and shall refer to one Person only; **Warrants shall be issued solely against persons who can reasonably be suspected to have been involved in an offence;** every Warrant granted for a continuous action shall expire no later than after one year and shall be renewed by another Judge in accordance with ordinary procedure.

**Comment by terrorist96** [comment]
Though I think this would be great in a utopia, I feel like it would cripple the judicial system and make prosecuting crimes nearly impossible unless the cop has video footage of the crime as it was happening.

**Comment by yopdog** [comment]
Is it really utopian to believe we can get witnesses to testify in court without forcing them? I think most witnesses would voluntarily be willing to cooperate with the court if asked. Paying the witnesses to testify is another option for getting them to cooperate if they are not really enthusiastic about it. That said, I think it should be obvious that subpoenas violate the rights of individuals to live their life without interference. To be forced to be somewhere at a certain time and take part in something you don't want basically amounts to slavery. For one day you no longer own yourself, the court owns you. I don't see how we could possibly justify this.

**Comment by terrorist96** [comment]
I don't disagree with the premise that you are deprived of your liberty by being subpoenaed. I don't know. It's a sticky situation.

**Comment by KacperZajc** [comment]
No one would take our legal system seriously.

**Comment by yopdog** [comment]
@KacperZajc I don't think worrying about what others may think of the legal system the main goal here. Liberland will be a country unlike any other and many people are already not taking it seriously. What we should be worrying about is creating laws that will give people the most freedom possible while at the same time protecting their rights.    

As for my suggestion I think we should be asking two sets of questions:

1) Are subpoenas justifiable under libertarian principals? Does forcing a witness to appear in court violate his right of self-ownership? How do we justify it?

2) Do we have to force witnesses to appear in court in order to get them to testify? What other ways do we have of getting them to cooperate without using force? Can it be done differently?     

Suggested reading:
https://mises.org/library/tyranny-government-courts-and-prisons by Murray N. Rothbard

**Comment by terrorist96** [comment]
I read your link and understand where you're coming from. I agree in principle, but don't really see a realistic workaround.

**Comment by yopdog** [comment]
Why would asking witnesses to voluntarily appear in court not be realistic?? What about the option of paying them? Would you seriously support the idea of a person going to jail for refusing to be the court's  property?

I really think you should be looking more closely at this. If you truly agree in principal, you would understand that this is not much different than redistribution of wealth. When a subpoena is issued it basically benefits one person (a defendant/prosecutor) at the direct expense of the person whom the subpoena was issued to. Instead of peacefully acquiring the service of that person, the subpoena allows for it to be stolen.

In other words, the philosophy of subpoenas can be summed up to the following: You have the right to life, liberty, and property so long as your property and time isn't too valuable to someone else. In the case where it is, you must hand over life, liberty, and property or face imprisonment. 

In my mind, that's a complete joke.

**Comment by terrorist96** [comment]
> Why would asking witnesses to voluntarily appear in court not be realistic?? What about the option of paying them? 

Where would the money come from?

> Would you seriously support the idea of a person going to jail for refusing to be the court's property?

It's a necessary evil in order to achieve justice.

**Comment by yopdog** [comment]
>Where would the money come from?

The money can come from the prosecutor or the defendant depending on who is asking the witness to testify his behalf. 

>It's a necessary evil in order to achieve justice.

You are following the same logic the is used to defend forced conscription or any other form of involuntary servitude. You could also make the same argument that forced military service is a necessary evil that everyone has to live with. What's more, by achieving justice this way you also create a new injustice. 

Also, explain to me how subpoenas fit in with provision §II.22. 

>§II.22. No law shall allow for the existence of any relationship between any legal or natural Person and any other Person which would amount to slavery or any other form of **involuntary servitude**; no conscription or any other form of **compulsory service** shall be introduced by any branch of the Public Administration.

I really don't see the problem with just not forcing witnesses to appear in court. An effective justice system requires the labor of many people. almost all that labor is supplied voluntarily through contracts. Why is the service of witnesses any different? Do we have to force judges to serve in order to have judges? Do we have to enslave lawyers in order to have them represent victims in court? What makes witnesses the exception?

**Comment by terrorist96** [comment]
> The money can come from the prosecutor or the defendant depending on who is asking the witness to testify his behalf.

What if it's free representation for a destitute person? That's more money the cash-strapped government would have to pay. 

> Do we have to force judges to serve in order to have judges? Do we have to enslave lawyers in order to have them represent victims in court? What makes witnesses the exception?

Judges and Lawyers are employed by the court, witnesses aren't. Witnesses are needed in order to obtain justice. Without their cooperation, they wouldn't be inclined to participate. A witness could be intimidated to not appear. 

How about if we allowed witnesses to be exempt from the land fee for that year as payment for their service to the court?

**Comment by yopdog** [comment]
>What if it's free representation for a destitute person? That's more money the cash-strapped government would have to pay.

That would still be better than using force against witnesses. That said, I never really supported the idea of the government supplying legal representation in the first place, but if it has to be this way I guess this is how it's going to work.  

>Witnesses are needed in order to obtain justice. Without their cooperation, they wouldn't be inclined to participate.

I understand that they are needed to obtain justice, but we need to remember that we are talking about Individuals with rights. We can't just force them to appear in court. We don't OWN their life. Therefore, the only moral way to get them to justify in court would be to ask them nicely and provide compensation if necessary. 

>A witness could be intimidated to not appear

Then lets not also intimidated them with imprisonment and fines if they don't appear.    

>How about if we allowed witnesses to be exempt from the land fee for that year as payment for their service to the court?

That would be the same as the government paying them outright to appear, but yes that's one option. Although I would prefer it if  the government wasn't involved in compensating witnesses.

**Comment by KacperZajc** [comment]
Im definitely against this idea.
It might lead to people being unjustly convicted.
Plus it will deter people who might want to set up companies in Liberland because of problems with potential litigations in case of dispute.

**Comment by yopdog** [comment]
@KacperZajc I must ask, how is this even be justified under libertarian principals to force a witness to appear in court? You are basically asking a person to do a compulsory service. This contradicts provision §II.22. and human rights in general.

The philosophy behind subpoenas is the same as the one for compulsory conscription. Being that if a person's labor or service is valuable enough, it can be stolen and taken away from him by force. If you truly believed that people *OWN* their lives, you would understand why I am concerned about this.

Addressing your concerns about the viability of doing away with subpoenas, I would point out that all we are doing here is moving the cost from the witness to the prosecutor or the defendant.
This is the great myth about forced labor/service, that somehow it saves on costs or makes things more efficient. All it does in reality is move the cost from the person who wants the service (the defendant/prosecutor) to the one providing it (a witness), violating human rights in the process.

My main suggestion is that witnesses be asked nicely for their time and be provided with compensation if necessary to encourage them to cooperate. It's their *life* and their *time* after all. 

Would this increase the cost of legal proceedings? Yes, but also asking a plumber to come into your home costs money, that doesn't mean you can enslave him. 

>It might lead to people being unjustly convicted.

I agree it could make finding witnesses who will testify a bit harder. Whatever effect this has, it will probably affect the prosecutor more since he has the burden of proof.

That said, you are only looking at one side of the picture. You have to look at the other side which is the rights of individuals to own their lives and spend their time as they wish. Forcing an innocent person to give up his time like that would be denying that he *owns* his life. You can't do that!

**Comment by KacperZajc** [comment]
I can get on board with financial compensation for the time spent in court if but we are definitely keeping this 'compulsory service'.

People want to be safe thereby need a well-functioning system of administration of justice. Without the right to compel one to appear in court, this system will not function well. It's just one of those small things that one has to agree to in order to live in a society. Since we are concerned here with the issue of domestic tranquillity, it falls under areas of competence of (even) a libertarian state. I dont think it is a big deal to appear in court but it is necessary for justice to be achieved.

**Comment by yopdog** [comment]
I'm sorry, but I really can't agree with what you are saying. You are looking at people as means to an end rather than an end in themselves. According to libertarian principals, you can't force a person to do something no matter how much you need his service. Even if you need him to testify at your trial you can't force him since he owns himself and not the other way around. 

Having said that, It seems to me like your main concern is with the  viability of having a well functioning justice system without subpoenas. My answer to that would for you to look at the countless ways we can get people to cooperate without using force. The absence of subpoenas will not mean that no one will testify. It will simply mean that you would have to use peaceful methods such as payment or just asking politely to get witnesses to give you their time and testify on your behalf. 

I believe most decent people will be willing to cooperate if asked to appear in court for a very modest compensation or none at all. The courts will function just fine since it would simply be an addition to the cost of justice. 

Again, you can't just look at this from the side of the court. People have plans for their time. Being forced to appear in court at a certain day can really mess up a person's schedule, it could cost him money! I try to look at it from both sides and what I believe is that we can both have a well-functioning justice system and respect people's rights at the same time. In essence, they are two sides of the same coin.

**Comment by yopdog** [comment]
If you are not convinced I suggest you read the following article by one of the best-known libertarian thinkers - Murray N. Rothbard
https://mises.org/library/tyranny-government-courts-and-prisons

**Comment by KacperZajc** [comment]
I am including this in #281 for further discussion.
